src/entities/mellowLpVault.ts

```diff
@@ -450,19 +455,25 @@ class MellowLpVault {
 
       try {
         await this.refreshWalletBalance();
-      } catch (_) {
+      } catch (error) {
+        Sentry.captureException(error);
+        Sentry.captureMessage('Wallet user balance failed to refresh after deposit');
         console.error('Wallet user balance failed to refresh after deposit');
       }
 
       try {
         await this.refreshUserDeposit();
-      } catch (_) {
+      } catch (error) {
+        Sentry.captureException(error);
+        Sentry.captureMessage('User deposit failed to refresh after deposit');
         console.error('User deposit failed to refresh after deposit');
       }
 
       try {
         await this.refreshVaultCumulative();
-      } catch (_) {
+      } catch (error) {
+        Sentry.captureException(error);
+        Sentry.captureMessage('Vault accumulative failed to refresh after deposit');
         console.error('Vault accumulative failed to refresh after deposit');
```

Comment:
Same question as above (throw new error pattern)

Reply:
Not sure if this place needs to rethrow the error. Whoever wrote this code considered this not to be the case and ignore the error completely. I am just tracking it with this PR. Perhaps it can be challenged once we have more information thanks to the tracking.